news/tests.py

```diff
@@ -38,6 +39,10 @@ def test_can_create_news_article(self):
         news_index.add_child(instance=article)
         self.assertEqual(article.heading, "Sample Article")
 
+    def bad_slug_returns_404(self):
+        response = self.client.get('/apps/cms/api/news/bad-slug/', format='json')
+        self.assertEqual(response.status_code, 404)
```

Comment:
codecov has officially lost it's mind if it's complaining that I am not testing the tests...

Reply:
Does it mean that test is never run?

Reply:
@RoyEJohnson, the Travis whisperer™️. That's exactly what's happening... I'll investigate.